cms/templates/index.html

```diff
@@ -443,10 +443,25 @@ <h3 class="course-title">${library_info['display_name']}</h3>
       <div class="notice notice-incontext notice-instruction notice-instruction-nocourses list-notices libraries-tab">
         <div class="notice-item">
           <div class="msg">
-            <div class="copy">
-              <p>${_("You don't have any content libraries yet.")}</p>
-            </div>
+              <h3 class="title">${_("Expecting to see a library already here?")}</h3>
+              <div class="copy">
+                  <p>${_('You will need to be added to the library in {studio_name} by the library creator. Please get in touch with the library creator or administrator for the specific library you are helping to author.').format(studio_name=settings.STUDIO_SHORT_NAME)}</p>
```

Comment:
Suggested wording: "The library creator must give you access to the library. Contact the library creator or administrator for the library you are helping to author."

NB -- I realize this is not parallel with the existing text for empty courses. I would like to make changes to the empty course text too.